opencensus/trace/span_context.py

```diff
@@ -156,8 +157,7 @@ def generate_span_id():
     :rtype: str
     :returns: 16 digit randomly generated hex trace id.
     """
-    span_id = uuid.uuid4().hex[:16]
-    return span_id
+    return '{:016x}'.format(random.getrandbits(64))
```

Comment:
What if the user makes explicit call to random.seed? Will this generate the same sequence of ids under the same seed?
If yes, potentially we want a separate random seed / generator (which is separate from user code).

Reply:
`random.seed()` effectively modifies the global state of your program's use of `random`. If a user is intentionally setting the `seed`, then isn't the expected behavior that the characteristics of the generator would change? Could certainly isolate it.